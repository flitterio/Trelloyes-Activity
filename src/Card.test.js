import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card Tests', () =>{

it('renders without crashing', ()=> {
    const div =document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders the title as expected', () =>{
    const tree = renderer
    .create(<Card name = "Card" title = "title" />)
    .toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders the content as expected', () =>{
    const tree = renderer
    .create(<Card name = "Card" content = "content" />)
    .toJSON();
    expect(tree).toMatchSnapshot();
});

});