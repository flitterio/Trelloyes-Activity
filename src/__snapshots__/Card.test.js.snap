// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Tests renders the content as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p>
    content
  </p>
</div>
`;

exports[`Card Tests renders the title as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title
  </h3>
  <p />
</div>
`;
